themes/jb/layouts/calendar/single.html

```diff
@@ -5,11 +5,8 @@
   <div class="container">
       <div class="content">
         <h1>{{.Title}}</h1>
-        Subscribe To The Calendar: <a href="{{.Site.Params.calendar.rss}}">iCalendar</a>
-
-        <iframe frameborder="0" scrolling="no" data-src="//www.google.com/calendar/embed?showTitle=0&amp;showNav=0&amp;showPrint=0&amp;showTabs=0&amp;showCalendars=0&amp;mode=AGENDA&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src={{.Site.Params.calendar.embedd}}&amp;color=%235229A3" width="100%" height="400px"></iframe>
-
-        <select name="ctz" id="ctz" style="width: 100%;">
+        <h2>Regular Live Show Times</h2>
```

Comment:
I was thinking about how these shows have two schedules: a live show schedule and a recorded show release schedule.

Not sure how much this header helps differentiate but wanted to try something.